package.json

```diff
@@ -52,7 +52,7 @@
     "dotenv": "^16.0.0",
     "fastify": "^4.0.0-rc.2",
     "generify": "^4.0.0",
-    "help-me": "^4.0.0",
+    "help-me": "^2.0.1",
```

Comment:
v4.0.0 should be good now.

Reply:
it isn't, see https://github.com/fastify/fastify-cli/pull/496. on windows it goes into an infinite loop that looks something like this 
![image](https://user-images.githubusercontent.com/20181/172407362-d72f45b3-09dc-4f95-9298-6e3017b71950.png)